src/chart/RechartsWrapper.tsx

```diff
@@ -12,18 +14,24 @@ export type RechartsWrapperProps = {
 };
 
 export const RechartsWrapper = forwardRef(
-  (
-    { children, width, height, className, style = {}, wrapperEvents = {} }: RechartsWrapperProps,
-    ref: Ref<HTMLDivElement>,
-  ) => (
-    <div
-      className={clsx('recharts-wrapper', className)}
-      style={{ position: 'relative', cursor: 'default', width, height, ...style }}
-      role="application"
-      {...wrapperEvents}
-      ref={ref}
-    >
-      {children}
-    </div>
-  ),
+  ({ children, width, height, className, style, wrapperEvents }: RechartsWrapperProps, ref: Ref<HTMLDivElement>) => {
+    const dispatch = useAppDispatch();
+    const innerRef = (node: HTMLDivElement | null) => {
+      dispatch(setContainer(node));
```

Comment:
Hmm, we probably shouldn't be dispatching an element into the store? But all good

Reply:
Correct, Redux will consider this an antipattern. I tried to only dispatch the domRect, and sizes (which is the correct Redux way) but then the tooltip will not react to resize events. We can solve that with Resize observer... Later.

Reply:
Ack, makes sense